aten/src/ATen/native/cuda/DepthwiseConv2d.cu

```diff
@@ -164,7 +168,11 @@ __global__ void conv_depthwise2d_backward_kernel(
 
 
 template <typename scalar_t, typename index_t=unsigned>
-__global__ void conv_depthwise2d_grad_weight_kernel(
+__global__ void
+#if __CUDA_ARCH__ == 620
+  __launch_bounds__(CUDA_NUM_THREADS, 1)
```

Comment:
Block size for this kernel's launches [comes from](https://github.com/pytorch/pytorch/blob/ebd727fc9f4c1f273b2395b9942cd2dc1d775e93/aten/src/ATen/native/cuda/DepthwiseConv2d.cu#L450-L454)
```C++
int getGradParamsNumThreads(int batchSize) {
  //warp per item in a batch, up to a maximum
  constexpr int MAX_BLOCK_SIZE = 256;
  return std::min(batchSize * C10_WARP_SIZE, MAX_BLOCK_SIZE);
}
```

The right thing to use is
```
__launch_bounds__(MAX_BLOCK_SIZE, 1)
```
But MAX_BLOCK_SIZE isn't globally available. So make the change below, then use
```
__launch_bounds__(GRAD_PARAMS_MAX_BLOCK_SIZE, 1)
```